alpha/ibm-object-storage-plugin/orderd-values.yaml

```diff
@@ -0,0 +1 @@
+values.yaml
```

Comment:
The file should be named `ordered-values.yaml` (mind the typo)

Reply:
Fixed the typo